src/Codeception/Module/WebDriver.php

```diff
@@ -416,6 +428,64 @@ protected function formatLogEntries(array $logEntries)
         return $formattedLogs;
     }
 
+    /**
+     * Turns an array of log entries into a HTML string containing only relevant errors.
+     * Each log entry is an array with the keys "timestamp", "level", and "message".
+     * See https://code.google.com/p/selenium/wiki/JsonWireProtocol#Log_Entry_JSON_Object
+     *
+     * @param array $logEntries
+     * @return string
+     */
+    protected function formatLogEntriesToHtml(array $logEntries)
```

Comment:
actually, why HTML? Can't this be also useful to have it inside console?
probably we shouldn't put view details and formatting inside the module?